src/handlers/lib/utility.js

```diff
@@ -399,10 +399,10 @@ const createParticipantTopicConf = (participantName, functionality, action, part
  *
  * @returns {object} - Returns newly created general topicConfig
  */
-const createGeneralTopicConf = (functionality, action, partition = 0, opaqueKey = 0) => {
+const createGeneralTopicConf = (functionality, action, partition = 0, opaqueKey = Uuid()) => {
```

Comment:
please add "key" parameter which defaults to null.
the opaqueKey should default to null